app/(app)/session/[session].tsx

```diff
@@ -1,24 +1,29 @@
 import { AntDesign, FontAwesome5, MaterialCommunityIcons } from '@expo/vector-icons';
 import { useTheme } from '@react-navigation/native';
+import { useQuery } from '@tanstack/react-query';
 import { Image } from 'expo-image';
+import * as Linking from 'expo-linking';
 import { Stack, useLocalSearchParams, useRouter } from 'expo-router';
 import React, { useState } from 'react';
-import { Pressable, StyleSheet, View, Alert, Share } from 'react-native';
+import { Alert, Pressable, Share, StyleSheet, View } from 'react-native';
 import Row from '../../../components/common/Row';
 import Space from '../../../components/common/Space';
 import StyledText from '../../../components/common/StyledText';
 import MainContainer from '../../../components/container/MainContainer';
-import { Sessions } from '../../../mock/sessions';
+import type { SessionForSchedule, Speaker } from '../../../global';
+import { getSessions, usePrefetchedEventData } from '../../../services/api';
 import { getSessionTimesAndLocation, getTwitterHandle, truncate } from '../../../util/helpers';
-import * as Linking from 'expo-linking';
 
 const Session = () => {
   const { colors, dark } = useTheme();
   const { slug } = useLocalSearchParams();
   const router = useRouter();
 
+  const { data: sessions } = useQuery({ queryKey: ['sessions'], queryFn: () => getSessions(50) });
+  const { schedule } = usePrefetchedEventData();
```

Comment:
On this one you could use `{{url}}/events/{{event_slug}}/sessions/{{session_slug}}` query. This is because I am noticing some sessions are not being found.